bundles/GoogleMarketingBundle/docs/README.md

```diff
@@ -3,7 +3,6 @@
 The `Marketing Settings` give you the possibility to configure marketing-specific settings, which are:
 
 - [Google Analytics](./05_Analytics.md)
-- Google Search Console
```

Comment:
mhhh, this and the other tab verification file instruction should be not removed, right?
it's not related to CSE

Reply:
The history looks right in the IDE. Maybe something todo with the treshhold like we discussed before the christmas party?

Reply:
the comment is unrelated to the conversation change about Search Console